samples/deployment/test-network/README.md

```diff
@@ -206,3 +206,31 @@ peer chaincode query -C mychannel -n ercc -c '{"Function": "queryListEnclaveCred
 E_ID=$(peer chaincode query -C mychannel -n ercc -c '{"Function": "queryListProvisionedEnclaves", "Args" : ["echo"]}' 2> /dev/null  | jq -r '.[0]')
 peer chaincode query -C mychannel -n ercc -c '{"Function": "queryEnclaveCredentials", "Args" : ["echo", "'${E_ID}'"]}'
 ```
+
+
+## Adding Blockchain Explorer
+
+Another way to look at the transactions and confirm how they are written to the ledger is adding [Hyperledger Explorer] (https://github.com/hyperledger/blockchain-explorer) to our test-network. To do so we will create a directory under test-network and we will copy some files (per the previous instructions) and directories from the network created.
+
+```bash
+cd $FPC_PATH/samples/deployment/test-network
+mkdir blockchain-explorer
+wget https://raw.githubusercontent.com/hyperledger/blockchain-explorer/main/examples/net1/config.json
+wget https://raw.githubusercontent.com/hyperledger/blockchain-explorer/main/examples/net1/connection-profile/test-network.json -P connection-profile
+wget https://raw.githubusercontent.com/hyperledger/blockchain-explorer/main/docker-compose.yaml
```

Comment:
Could we also do that in `setup.sh`. Also the folder `blockchain-explorer` should be added to gitignore.

Reply:
Lastest version of this PR extends setup.sh with the BE installation.